playbooks/install_vision_dependencies.yml

```
@@ -12,15 +12,17 @@
         state: latest
         update_cache: true
       become: true
-      register: apt_output
       tags: always
 
-    - name: install pytorch
+    - name: install pytorch & create venv
       pip:
         name: https://github.com/pyronear/pyro-vision/releases/download/v0.1.1/torch-1.7.0a0-cp37-cp37m-linux_armv7l.whl
+        virtualenv: /home/pi/python_scripts/venv
+        virtualenv_command: python3 -m venv
```

```suggestion
        virtualenv_command: python3 -m venv
```
In doing it this way, it creates en venv and install pytorch in it ?    And what's the name of the venv then ? "python_scripts" ?
Sorry, I am not knwoledgeable with the ansible way of doing it :)

---

FYI: In my last commit (just now), I regrouped the installation of pytorch and torchvision

To answer your question, you can take a look at the [Ansible pip](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html) documentation. TLTR:
- in `virtualenv`: you give the path to the venv. if it exists, great. If not Ansible will create it for you but in that case you need to specify the `virtualenv_command`
If the venv already exists, you just need to specify the path to it ! 

The name of the virtual env is `venv` as specified in the path to the virtual env (which is in the folder `python_scripts`)

---

thanks !